sycl/test-e2e/Graph/Threading/update.cpp

```diff
@@ -0,0 +1,84 @@
+// REQUIRES: level_zero, gpu, TEMPORARY_DISABLED
```

Comment:
Would it make sense to temporary postpone adding those tests until they are actually supported?
It would simplify this PR and could improve tests quality, because reviewers could skip those tests right now. Moreover, there is no way to ensure that those can be even compiled, because `LIT` will completely skip them.

Reply:
Makes sense. Addressed by https://github.com/reble/llvm/pull/266